karamel-core/src/test/java/se/kth/karamel/cookbook/metadata/TestKaramelizedCookbook.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
I had moved these tests into other classes, I guess it has appeared in the merge again.